charts/hono/example/jaeger/default-sampling-strategies.json

```diff
@@ -0,0 +1,64 @@
+{
+  "service_strategies": [
+    {
+      "service": {{ printf "%s-service-device-registry" .Release.Name | quote }},
+      "type": "probabilistic",
+      "param": 1.0,
+      "operation_strategies": [
```

Comment:
When doing a local deployment with
```
helm install --dependency-update -n hono --set honoImagesType=quarkus --set jaegerBackendExample.enabled=true --set adapters.amqp.enabled=false --set adapters.http.enabled=false --set prometheus.createInstance=true --set grafana.enabled=false --set dataGridExample.enabled=true eclipse-hono hono/
```
the `eclipse-hono-service-device-registry-0` pod fails to start.
There is the following exception:
````
13:22:19.539 [main] ERROR o.s.boot.SpringApplication - Application run failed
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'application': Unsatisfied dependency expressed through method 'setVerticles' parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'amqpServer': Unsatisfied dependency expressed through method 'setTracer' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'getTracer' defined in org.eclipse.hono.deviceregistry.file.ApplicationConfig: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [io.opentracing.Tracer]: Factory method 'getTracer' threw exception; nested exception is java.lang.NoClassDefFoundError: com/google/gson/JsonSyntaxException
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.resolveMethodArguments(AutowiredAnnotationBeanPostProcessor.java:768)
[...]
	at org.eclipse.hono.deviceregistry.file.Application.main(Application.java:110)
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'amqpServer': Unsatisfied dependency expressed through method 'setTracer' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'getTracer' defined in org.eclipse.hono.deviceregistry.file.ApplicationConfig: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [io.opentracing.Tracer]: Factory method 'getTracer' threw exception; nested exception is java.lang.NoClassDefFoundError: com/google/gson/JsonSyntaxException
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.resolveMethodArguments(AutowiredAnnotationBeanPostProcessor.java:768)
[...]
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.resolveMethodArguments(AutowiredAnnotationBeanPostProcessor.java:760)
	... 20 common frames omitted
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'getTracer' defined in org.eclipse.hono.deviceregistry.file.ApplicationConfig: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [io.opentracing.Tracer]: Factory method 'getTracer' threw exception; nested exception is java.lang.NoClassDefFoundError: com/google/gson/JsonSyntaxException
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:658)
[...]
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.resolveMethodArguments(AutowiredAnnotationBeanPostProcessor.java:760)
	... 37 common frames omitted
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [io.opentracing.Tracer]: Factory method 'getTracer' threw exception; nested exception is java.lang.NoClassDefFoundError: com/google/gson/JsonSyntaxException
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:185)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:653)
	... 50 common frames omitted
Caused by: java.lang.NoClassDefFoundError: com/google/gson/JsonSyntaxException
	at io.jaegertracing.Configuration$SamplerConfiguration.createSampler(Configuration.java:392)
	at io.jaegertracing.Configuration.getTracerBuilder(Configuration.java:231)
	at io.jaegertracing.Configuration.getTracer(Configuration.java:253)
	at io.jaegertracing.tracerresolver.internal.JaegerTracerFactory.getTracer(JaegerTracerFactory.java:24)
	at io.jaegertracing.tracerresolver.internal.JaegerTracerFactory.getTracer(JaegerTracerFactory.java:21)
	at io.opentracing.contrib.tracerresolver.TracerResolver.getFromFactory(TracerResolver.java:190)
	at io.opentracing.contrib.tracerresolver.TracerResolver.resolveTracer(TracerResolver.java:123)
	at io.opentracing.contrib.tracerresolver.TracerResolver.resolveTracer(TracerResolver.java:79)
	at org.eclipse.hono.deviceregistry.file.ApplicationConfig.getTracer(ApplicationConfig.java:98)
	at org.eclipse.hono.deviceregistry.file.ApplicationConfig$$EnhancerBySpringCGLIB$$dbe47f89.CGLIB$getTracer$5(<generated>)
	at org.eclipse.hono.deviceregistry.file.ApplicationConfig$$EnhancerBySpringCGLIB$$dbe47f89$$FastClassBySpringCGLIB$$6c46c8f0.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:331)
	at org.eclipse.hono.deviceregistry.file.ApplicationConfig$$EnhancerBySpringCGLIB$$dbe47f89.getTracer(<generated>)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154)
	... 51 common frames omitted
Caused by: java.lang.ClassNotFoundException: com.google.gson.JsonSyntaxException
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	... 70 common frames omitted
````

It seems the file-based device registry component is missing the `com.google.code.gson` jar which is needed when using the `RemoteControlledSampler` (it's imported in the used `io.jaegertracing.internal.samplers.HttpSamplingManager`).

That error only occurs when using the standard Hono 1.7.1 images. When testing with local snapshot images (built from current master), there is no such error.